corehq/apps/analytics/tasks.py

```diff
@@ -148,6 +148,13 @@ def _send_form_to_hubspot(form_id, webuser, cookies, meta):
 
 
 @task(queue='background_queue', acks_late=True, ignore_result=True)
+def update_hubspot_properties(webuser, properties):
+    vid = _get_user_hubspot_id(webuser)
+    if vid:
+        _track_on_hubspot(webuser, properties)
```

Comment:
Unfortunately Hubspot doesn't have that functionality, and I've wondered about why we do that in the past. 
@NoahCarnahan do you remember why you did the `_get_user_hubspot_id(webuser)` when you first integrated hubspot? I think everything else has stemmed from code you wrote initially.